xml/obs_best_practice_staging_workflow.xml

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 version="5.1" xml:id="staging_how_to"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Staging</title>
+  <para></para>
+  <sect2>
+    <title>Creating a Staging</title>
+    <para>
+      At the moment, the Staging is only accessible by hand.
+      "<emphasis>/staging_workflows/new?project=openSUSE:Factory</emphasis>"
+      The creation of a Staging will automatically create two staging projects
+      as a subproject of the main project. Before create we need to select a
+      group of managers, they will be in charge of assign requests to the
+      staging projects and also exclude requests from the Staging.
+    </para>
+    <figure>
+     <title>Creating a Staging for openSUSE:Factory</title>
+     <mediaobject>
+      <imageobject>
+       <imagedata fileref="staging_workflow_create.png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+    <note>
+      <para>
+        An Admin should previously create the manager groups.
+      </para>
+    </note>
+  </sect2>
+  <sect2>
+    <title>Start using it</title>
+    <para>
+      In this view, we could find all the staging projects with a request
+      associated and also if the requests have some problems.
+    </para>
+    <figure>
+     <title>Staging Show screen</title>
+     <mediaobject>
+      <imageobject>
+       <imagedata fileref="staging_workflow_show.png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+    <itemizedlist>
+      <listitem>
+        <para>
+          <emphasis>Managers:</emphasis>
+          Shows the group that are the Staging Mangers
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis>Empty projects:</emphasis>
+          Staging projects without and requests assigned.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis>Backlog:</emphasis>
+          List of requests that can be assigned to a staging project.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis>Ready:</emphasis>
+          List of requests that were in the backlog and have a review accepted.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis>Ignored:</emphasis>
+          List of requests excluded from this Staging.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect2>
+  <sect2>
+    <title>Configuration</title>
+    <para>
+      The configuration of a Staging is very intuitive. We are able to
```

Comment:
```suggestion
      It is possible to
```

If it is intuitive or not is not our responsibility to judge :smile: (and also not part of a documentation)